classes/Server.py

```diff
@@ -35,6 +36,8 @@ def start(self, host, port):
             return
         logging.info('Launching server ...')
         self.started = True
+        logging.info('loading Plugins ...')
+        plugin.load_plugins()
```

Comment:
While there is nothing wrong with the line in particular, it does look strange to say the least. `plugin` would suggest that it is the variable `plugin` -  having plugin.load_plugins() (singular variable name with plural method name) does not make it better.